checker/check-request-property-min-length-updated.go

```diff
@@ -0,0 +1,95 @@
+package checker
+
+import (
+	"fmt"
+
+	"github.com/tufin/oasdiff/diff"
+)
+
+func RequestPropertyMinLengthUpdatedCheck(diffReport *diff.Diff, operationsSources *diff.OperationsSourcesMap, config Config) Changes {
```

Comment:
this checker didn't exist, added it. not sure about what we want to classify as breaking here but happy to follow-up in another PR

Reply:
Good catch.
Increasing min in request is breaking, see below.

Reply:
Please add the checker to defaultChecks